linker-interface/shared/src/main/scala/org/scalajs/linker/interface/Semantics.scala

```diff
@@ -178,6 +179,43 @@ object Semantics {
         replacement: String): RuntimeClassNameMapper = {
       regexReplace(regex.pattern, replacement)
     }
+
+    private[interface] implicit object RuntimeClassNameMapperFingerprint
+        extends Fingerprint[RuntimeClassNameMapper] {
+
+      override def fingerprint(mapper: RuntimeClassNameMapper): String = {
+        mapper match {
+          case KeepAll => "KeepAll"
+          case DiscardAll => "DiscardAll"
+          case RegexReplace(pattern, flags, replacement) =>
+            new FingerprintBuilder("RegexReplace")
+              .addField("pattern", pattern)
+              .addField("flags", flags)
+              .addField("replacement", replacement)
+              .build()
+          case AndThen(first, second) =>
```

Comment:
I think there needs to be a blank line before this. Also in other spots.

Reply:
Fixed in eb6e308689224c3bbcc9c059e10fa1f02bee3acb